pkg/kclient/operators.go

```diff
@@ -6,6 +6,16 @@ import (
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	ErrNoSuchOperator = odoError("Could not find specified operator")
+)
+
+type odoError string
```

Comment:
you don't need this actually. you can directly do `ErrNoSuchOperator = errors.New("Could not find specified operator")`, unless you are planning to match the `odoError` type in the code.

Reply:
At the moment, I don't know if I'll be using it to match anything. This might be useful in future. Anyway, I've changed it.